chandra_aca/aca_image.py

```diff
@@ -499,7 +507,9 @@ def _flicker_update_vectorized(self, dt):
         rand_ampls = np.random.uniform(0.0, 1.0, size=len(idxs))
         rand_times = np.random.uniform(0.0, 1.0, size=len(idxs))
 
-        for idx, rand_time, rand_ampl in zip(idxs, rand_times, rand_ampls):
+        for idx, rand_time, rand_ampl in zip(
+            idxs, rand_times, rand_ampls, strict=False
```

Comment:
Use `strict=True`. If these are not all the same length then there is a serious problem.